frontend/views/containers/modals/SignUp.vue

```diff
@@ -147,13 +147,13 @@ export default {
           // 1. begin monitoring the contracts for updates via the pubsub system
           // 2. add these contracts to our vuex state
           for (const contract of [user, mailbox]) {
-            await this.$store.dispatch('syncContractWithServer', contract.hash())
+            await sbp('state/vuex/dispatch', 'syncContractWithServer', contract.hash())
```

Comment:
Could you explain why are we vuex (this.$store) is being migrated to sbp (and calling the selector vuex)?

Reply:
That's a long debate. From my understanding at the beginning

Reply:
That's a long debate. From my understanding, sbp should be used for every action that should be saved locally and/or synchronized with the server and users. The vuex store is used for  action with no recorded state, for example open the menu. Maybe greg want also to replace completely the vuex store with sbp.

Reply:
Well:

1. We might get rid of vuex completely in the future (and replace with a 100% sbp-based system), because we might switch to a different framework, see #587
2. Using `sbp` to call out to store modifications is the "right way" to do things, not because "I said so", but because SBP makes the code cleaner, improves debugging and logging, etc.
3. By using SBP it makes it possible to easier to write tests in certain situations, since selectors can be swapped out. For example, this is how we make `handleEvent` work in `test/backend.js` (by simulating it with a simplified version)

Reply:
Also, (4) SBP is really versatile, and works in many situations to do many different things, so it is already used all over the codebase. For this reason, it becomes simpler when writing code to just think in terms of SBP as much as possible. There are fewer concepts to deal with. I.e. it is simpler to think in terms of SBP, than to think in terms of SBP, and Vuex, and [this, that and the other]

Reply:
Now, calling the selector 'state/vuex/dispatch' is a reminder that when we will remove vuex completely we need to check this event.

Reply:
Thank you guys for the clarification. This intel is too good to be lost in a PR. It would be nice to add this info as documentation. Maybe at `model/state.js` it would be a good place.

Reply:
I'm not sure that `model/state.js` is the right place to put all this documentation... however maybe it could go somewhere in the `docs/` folder?